### PR TITLE
handle missing 'device' and 'client' in pinger logs

### PR DESCRIPTION
**Pull request by azimnacho**

### CODE REVIEW

File: T3Viewer/static/list.js [hunk 1]
```diff
@@ -291,6 +291,12 @@ function createTitleBar() {
     return tr;
 }
 
+Object.prototype.notIn = function() {
+    for(var i=0; i<arguments[0].length; i++)
+       if(arguments[0][i] == this) return false;
+    return true;
+}
+
```
Comment (jvilhuber) [inline T3Viewer/static/list.js:299]: Never seen this (Object.prototype). What does it do? Got a link I can read up on?

Comment (azimnacho) [inline T3Viewer/static/list.js:299]: stolen originally from a stackoverflow to implement an In function.
http://www.w3schools.com/js/js_object_prototypes.asp

Comment (jvilhuber) [inline T3Viewer/static/list.js:299]: Ah cool.

**Comment by jvilhuber** [comment]
Looks good.

**Pull request merged**